### ISSUE TITLE
Show label for new story button for screen reader

### ISSUE THREAD

**Issue by theachoem** [theachoem/storypad#419]
Got this issue from reddit:

```

Hey dev, thank you so much for sharing this app with us. The idea sounded cool, so I tried it. The app works very well, but as a screen reader user, I have some feedback.

The button to create a new entry is not labelled, so are the many formatting options. It means my screen reader will just say "button." As you see, it is not enough information for a nonvisual mobile user to understand what the button does. So it will be cool if you could make sure to label the buttons. If you want to read more about the importance of labelling the buttons correctly, please see the link below.

https://support.google.com/accessibility/android/answer/7158690?hl=en

I hope you fix this minor issue. Thank you so much for reading!
```
